src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -68,6 +68,20 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>NumberOfDaysFromNextDonationDate__c</fullName>
+        <defaultValue>0</defaultValue>
```

Comment:
As per discussion on W-038890, this new field should have a default value of 3.

Reply:
The API name has been changed to `NextDonationDateMatchRangeDays__c`. 
@sam-knox , can you please advise if the field label and description/help text should be changed?